operator/src/main/resources/application.properties

```diff
@@ -41,12 +41,20 @@ ingresscontroller.max-ingress-connections=108000
 ingresscontroller.peak-connection-percentage=100
 
 # Enables haproxy option contstats in the kas ingress so that stats are reported live rather than at connection close (RFE-3007 provide a proper mechanism to enable this option).
-ingresscontroller.ingress-container-command=/usr/bin/bash,-c,awk '{print $0} /^defaults$/ {print \"  option contstats\"}' < $TEMPLATE_FILE > /tmp/haproxy-config.template; exec /usr/bin/openshift-router --v=2 --template /tmp/haproxy-config.template
+ingresscontroller.ingress-container-command=/usr/bin/bash,-c,awk '{print $0} /^defaults$/ {print \"  option contstats\"}' < $TEMPLATE_FILE > /tmp/haproxy-config.template; exec /usr/bin/openshift-router --v=${ingresscontroller.router-verbosity} --template /tmp/haproxy-config.template --commit-interval=${ingresscontroller.commit-interval} --blueprint-route-pool-size=${ingresscontroller.blueprint-route-pool-size} --max-dynamic-servers=${ingresscontroller.max-dynamic-servers}
 # Disconnect established connections after a haproxy reconfiguration (NE-879 should eliminate the need for this)
 ingresscontroller.hard-stop-after=5s
 # Coalesce up-to reloadInterval worth of haproxy updates.  This prevents a flurry of haproxy restarts (1 per OpenShift Route) as each kafka
 # instance is created (NE-879 should eliminate the need for this).
 ingresscontroller.reload-interval-seconds=60
+ingresscontroller.router-verbosity=2
+
+# HA proxy dynamic config manager
+ingresscontroller.dynamic-config-manager=true
+# the following three variables are meaningful when the dynamic-config-manager=true
+ingresscontroller.commit-interval=9223372036854775807ns
```

Comment:
If we want to set it as `indefinite`, why can't we set it as `9223372036854775807h`?

Reply:
overflow? Presumably `ingresscontroller.commit-interval` is converted to a number of ns internally. Specifying it in NS is already almost 300 years

Reply:
> Specifying it in NS is already almost 300 years

Oh really! haha! Then that's good